requirements/lit/docs.txt

```diff
@@ -4,4 +4,6 @@ ipython[notebook]
 ipython_genutils
 
 sphinx-autobuild
-https://github.com/PyTorchLightning/lightning_sphinx_theme/archive/master.zip#egg=pt-lightning-sphinx-theme
+pt-lightning-sphinx-theme @ https://github.com/Lightning-AI/lightning_sphinx_theme/archive/master.zip
```

Comment:
The theme was installed correctly. We install the same theme the same way for PL docs.

Reply:
yep, it's the same but wasn't the same way. But that doesn't matter.